ssg/build_cpe.py

```diff
@@ -22,16 +27,16 @@ class CPEDoesNotExist(Exception):
 
 class ProductCPEs(object):
     """
-    Reads from the disk all the yaml CPEs related to a product
-    and provides them in a structured way.
+    This class serves as a structure for CPE items during the build process.
+    It allows adding and retrieving of items.
     """
 
     def __init__(self):
 
-        self.cpes_by_id = {}
-        self.cpes_by_name = {}
-        self.product_cpes = {}
-        self.platforms = {}
+        self.cpes_by_id = {} # CPE items indexed by internal ID
+        self.cpes_by_name = {} # CPE items indexed by CPE name
+        self.product_cpes = {} # CPE items which define applicability of a product indexed by internal ID
+        self.platforms = {} # CPE AL platforms indexed by internal ID
```

Comment:
It is interesting that a class `ProductCPEs` has an attribute `product_cpes` and a bunch of other attributes - there is a name clash going on. Also I don't believe that those comments are useful - why not to add the `_by_id` suffix to `product_cpes` and to `platforms`?